raiden/transfer/state.py

```diff
@@ -304,6 +305,15 @@ class PendingWithdrawState:
     recipient_metadata: Optional[AddressMetadata] = None
 
 
+@dataclass
+class CoopSettleState:
+    total_withdraw_participant: WithdrawAmount
+    total_withdraw_partner: WithdrawAmount
```

Comment:
The usual naming pair is `our` (instead of `participant`) and `partner`.

Reply:
I see now that these are not "our" and "our partner" but rather the participant initiating the coop settle and the participant's partner. If we don't find better names, we should at least add a short comment.

Reply:
Maybe... `total_withdraw_initiator`?

Reply:
Went with the initiator and left the other as partner.